### ISSUE TITLE
Go imports need updating after switching to MiniShift organization

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#146]
`go build` fails with below error.

```
$ go build
gen_help_text.go:20:2: cannot find package "github.com/jimmidyson/minishift/cmd/minikube/cmd" in any of:
	/Users/lmohanty/code/gocode/src/github.com/MiniShift/minishift/vendor/github.com/jimmidyson/minishift/cmd/minikube/cmd (vendor tree)
	/usr/local/go/src/github.com/jimmidyson/minishift/cmd/minikube/cmd (from $GOROOT)
	/Users/lmohanty/code/gocode/src/github.com/jimmidyson/minishift/cmd/minikube/cmd (from $GOPATH)
```

**Comment by hferentschik** [comment]
The beauty of Go :-( This absolute import seem odd to me. Either way, this is something we should fix asap, on master as well as the _cluster_up_ branch.

**Comment by gbraad** [comment]
I was able to compile using `make` after doing some renames; PR #152

**Comment by hferentschik** [comment]
Fixed via pull request #152
